src/main/webapp/src/app/schedule-page/schedule-page.component.ts

```diff
@@ -23,11 +22,7 @@ import {FormControl} from '@angular/forms';
     '../common/growpod-page-styles.css',
   ],
 })
-
 export class SchedulePageComponent implements OnInit {
-  date = new FormControl(new Date());
```

Comment:
why removing this?

Reply:
This was moved to be part of the datepicker component